src/Microsoft.Diagnostics.Monitoring.WebApi/ProfilerChannel.cs

```diff
@@ -30,7 +31,13 @@ public async Task SendMessage(IEndpointInfo endpointInfo, IProfilerMessage messa
         {
             if (message.Payload.Length > MaxPayloadSize)
             {
-                throw new ArgumentException(nameof(message));
+                throw new ArgumentException(
```

Comment:
NOTE: When making this `class` public it appears to enable additional analyzers. Specifically an analyzer that detects if `ArgumentException` ctors are being used incorrectly (the first argument should be the message, NOT the argument name). I've found other instances of this issue throughout our code but that should be a separate cleanup effortt.